service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderServiceTest.java

```diff
@@ -27,16 +27,13 @@
 import static uk.gov.hmcts.reform.fpl.enums.ActionType.SEND_TO_ALL_PARTIES;
 import static uk.gov.hmcts.reform.fpl.model.common.DocumentReference.buildFromDocument;
 import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createHearingBookingDynmaicList;
-import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createHearingBookings;
 import static uk.gov.hmcts.reform.fpl.utils.DocumentManagementStoreLoader.document;
 
 class CaseManagementOrderServiceTest {
     private static final LocalDateTime NOW = LocalDateTime.now();
     private final Time time = () -> NOW;
-    private static final LocalDateTime DATE = LocalDateTime.of(2018, 2, 12, 9, 30);
 
-    private CaseManagementOrderService service = new CaseManagementOrderService(time,
-        new DateFormatterService(), new HearingBookingService());
+    private CaseManagementOrderService service = new CaseManagementOrderService(time, new HearingBookingService());
```

Comment:
Still not sure why we haven't autowired in an instance of time, would mean that you don't need to manually create CaseManagementOrderService